Frameworks/Foundation/NSOperation.mm

```diff
@@ -84,11 +89,38 @@ - (void)_checkReady {
     }
 }
 
+- (void)_dispatchCompletionBlock {
+    dispatch_queue_t completionQueue = _completionQueue;
+    if (!completionQueue) {
+        // It is legal to run an NSOperation without an operation queue.
```

Comment:
>It is legal to run an NSOperation without an operation queue. [](start = 11, length = 61)

If iOS does not allow running NSOperation without an operation queue, do we have not be this nice?